### PR TITLE
Create list view layout

### PR DESCRIPTION
**Pull request by donbasilpeter-aot**
# Issue Tracking

JIRA: 
Issue Type: FEATURE
[Create List view Layout](https://aottech.atlassian.net/browse/FWF-2264)

# Changes
added list view along with the card view layout in the task component

# How has the change been tested?

# Screenshots (if applicable)
image-20230627-033554
image-20230627-033732

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+  const reqData = useSelector((state) => state.bpmTasks.listReqParams);
+  const firstResult = useSelector((state) => state.bpmTasks.firstResult);
+  const [followUpDate, setFollowUpDate] = useState(null);
+  const [dueDate, setDueDate] = useState(null);
+  const [showModal, setModal] = useState(false);
+  const [isEditAssignee, setIsEditAssignee] = useState(false);
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  useEffect(() => {
+    console.log(task, taskId);
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js:47]: remove console

Comment (abhilash-aot) [reply forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js:47]: @fahad-aot  Please check any comments coming to this PR and resolve

**Review by abhilash-aot — APPROVED** [review]
Taking this to new Task List branch and other changes can go in there and then merge to develop @fahad-aot , also update to latest develop code

File: forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js (hunk 1 above)
Comment (fahad-aot) [reply forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js:47]: okay

File: forms-flow-web/src/actions/actionConstants.js [hunk 2]
```diff
@@ -123,6 +123,8 @@ const ACTION_CONSTANTS = {
   RELOAD_TASK_FORM_SUBMISSION: "RELOAD_TASK_FORM_SUBMISSION",
   BPM_TASK_LIST_ACTIVE_PAGE: "BPM_TASK_LIST_ACTIVE_PAGE",
   DEPLOYMENT_LIST: "DEPLOYMENT_LIST",
+  SELETED_TASK_VARIABLES:"SELETED_TASK_VARIABLES",
```
Comment (AbijithS-aot) [inline forms-flow-web/src/actions/actionConstants.js:126]: check spelling

File: forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+  updateBPMTask,
+} from "../../../apiManager/services/bpmTaskServices";
+import { setBPMTaskDetailUpdating } from "../../../actions/bpmTaskActions";
+//import UserSelection from "./UserSelection";
+import UserSelectionDebounce from "./UserSelectionDebounce";
+import SocketIOService from "../../../services/SocketIOService";
+import { useTranslation } from "react-i18next";
+
+const TaskHeaderListView = React.memo(({task,taskId,groupView = true}) => {
+  // const task = useSelector((state) => state.bpmTasks.taskDetail);
```
Comment (AbijithS-aot) [inline forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js:30]: remove unused codes

**Comment by abhilash-aot** [comment]
Closing the PR as the branch is cloned and new PR will be given from taht

**Pull request closed without merging**